### PR TITLE
fix: update windows ci description to be correct

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: .github/workflows/integration-windows.yml [hunk 1]
```diff
@@ -28,6 +28,6 @@ jobs:
         fetch-depth: 0
 
     # TODO(aaron-prindle) skip integration tests for doc only changes on Windows.  Figure out the proper syntax, etc.
-    - name: Run windows unit tests and integration tests
+    - name: Run skaffold unit tests on windows
```
Comment (tejal29) [inline .github/workflows/integration-windows.yml:31]: do you want to use the same packages as darwin integration test?
```
  go test -short -timeout 60s  ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
```

Comment (aaron-prindle) [reply .github/workflows/integration-windows.yml:31]: I can update this to be the same as `darwin`.  For more context here, this is the same as what was in the travis.yml setup for windows as is currently:
```
    - os: windows
      name: "Windows unit"
      script:
        - go test -short -timeout 60s ./...
```

Comment (aaron-prindle) [reply .github/workflows/integration-windows.yml:31]: I've updated the windows ci tests to do a smaller subset of unit tests (the same as `darwin` as suggested)

Comment (tejal29) [reply .github/workflows/integration-windows.yml:31]: ah. I wonder how things were working before. 🤷

**Review by gsquared94 — APPROVED** [review]

**Pull request merged**